content/learning-paths/smartphones-and-mobile/android_webgpu_dawn/1-webgpu-fundamentals.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Introduction to WebGPU
+weight: 2
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## What is WebGPU?
+
+WebGPU is the successor to [WebGL](https://www.khronos.org/webgl/wiki/), a well adopted modern API standard for interfacing with GPUs. WebGPU provides better compatibility with modern GPUs, support for general-purpose GPU computations, faster operations, and access to more advanced GPU features. It is designed to provide a _unifed access_ to GPUs, agnostic to GPU vendors and Operating System the application runs on. WebGPU is essentially a Render Hardware Interface built on top of various backends APIs like Vulkan, DirectX, Metal depending on OS/platform. This duplicated development effort is made once by the web browsers and made available to us through the webgpu.h header they provide.
+
+![WebGPU High Level View](images/webgpu_highlevel.png "WebGPU High Level View")
+
+## Why WebGPU?
+
+WebGPU has been designed taking into learnings from older standards like WebGL, OpenGL etc. and is the only graphics API that benefits from
```

Comment:
"WebGPU takes learnings from older standards like WebGL and OpenGL and introduces new features that provide developers with a modern graphics standard for web and native applications. WebGPU is the only graphics standard that provides:

* A reasonable level of abstraction
* Good performance
* Cross-platform compability
* Backing from the W3C Standards group
* Future-proof design"

Reply:
Thanks, updated